### PR TITLE
[FEATURE] First try with pydata theme

### PR DESCRIPTION
**Pull request by jpfeuffer**

### CODE REVIEW

**Comment by jpfeuffer** [comment]
image

**Comment by jpfeuffer** [comment]
I envision having different docs on the main header:
- User guide
- Contributors guide
- API docs
- Link to C++ docs

**Comment by timosachsenberg** [comment]
any update here?

**Comment by jpfeuffer** [comment]
@timosachsenberg Still waiting for feedback from people

**Comment by matteopilz** [comment]
300 - 1000s are still fine I would say, Is that with building everything or only files changed?

**Comment by jpfeuffer** [comment]
Everything. We can also try some of the suggestions in the link from the issue: https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/performance.html#build-performance-and-size

Let me know if you like it in general. I still want to do quite some changes but the overall restructuring is finished.

**Comment by timosachsenberg** [comment]
I think it is great if it gets the side panel back.

**Comment by jpfeuffer** [comment]
It has. You need to click on a top bar link first, since the sidebar contents depend on this.

**Comment by matteopilz** [comment]
I also like it. If we exclude the _autosummary folder from the sidebar, the building time will probably be shorter.

**Comment by timosachsenberg** [comment]
ah ok and the top bar would not contain that many entries right? is there a working version up somewhere to get an idea how this would look like for the user?

**Comment by jpfeuffer** [comment]
> Hmm unfortunately pydata is really really slow with autosummary: [pydata/pydata-sphinx-theme#889](https://github.com/pydata/pydata-sphinx-theme/issues/889)
> 
> From 300 to 1000s build time on RTD. https://pyopenms.readthedocs.io/en/pydata-theme/index.html

Yep. See the second link in my comment.

**Comment by matteopilz** [comment]
Did you test it without autosummary?

**Comment by jpfeuffer** [comment]
You mean completely without autosummary? No, we need it for the API.
I did not test the autosummary exclusion from the sidebar yet.

Of course, on a local computer it is usually relatively fast. And once the autosummary is generated it is even faster.
But a full build on RTD is slow.

**Comment by matteopilz** [comment]
I meant excluding it from the sidebar.

**Comment by timosachsenberg** [comment]
- navigation structure is duplicated in sidebar / main text
image

- seems to render  slower on my PC (8k display). Can someone confirm?

**Comment by jpfeuffer** [comment]
a) was always the case. we can hide the TOC in the text.
b) might also be related to caching? but I would not be surprised if it is a bit slower. It is not very noticeable for me though.

**Comment by timosachsenberg** [comment]
some images are not drawn
https://pyopenms.readthedocs.io/en/pydata-theme/user_guide/ms_data.html

File: docs/source/img/DFPIANGER_isoDistribution.png
Comment (timosachsenberg) [inline docs/source/img/DFPIANGER_isoDistribution.png:1]: There are a lot of removed images. That’s fine right?

Comment (jpfeuffer) [reply docs/source/img/DFPIANGER_isoDistribution.png:1]: They were moved (into their respective subfolder), not removed. So, yes, that is fine.

Comment (jpfeuffer) [reply docs/source/img/DFPIANGER_isoDistribution.png:1]: You need to check the file summary of the whole PR. Not that single commit. I did it in two commits.

**Comment by jpfeuffer** [comment]
I could speed up generation by removing tocs from the API docs. Not ideally but also not too bad. Will merge

**Comment by timosachsenberg** [comment]
hmm I get a huge left border with adds now
image

**Comment by timosachsenberg** [comment]
can the "User guide" be enabled by default and the sections expanded?

**Comment by jpfeuffer** [comment]
I think it has to have a landing page.
Yes to can be expanded. See conf.py.
Ethical ads placement is default. See https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/readthedocs.html#add-ethical-advertisements-to-your-sidebar
on how to change.

**Comment by timosachsenberg** [comment]
- I think if expanded the adds are not a problem. maybe than landing page is also not an issue

**Pull request merged**